### DISCUSSION TITLE
How does DIA-NN treat Unfragmented Precursors in MS2?

### DISCUSSION THREAD

**Discussion by MHBI2018** [vdemichev/DiaNN#801] [General]
Hi,

Do partially fragmented or unfragmented precursors in MS2 spectra contribute to the identification rate or the scoring in any way?

Thank you

**Comment by vdemichev** [comment]
Yes, they do, but their effect performance-wise is quite insignificant usually

**Reply by MHBI2018** [reply]
Thank you @vdemichev,

I understand that a lot of factors go into the peptide scoring, but do unfragmented precursors influence identification rates in any way?

**Reply by vdemichev** [reply]
Yes
